lib/rest/utils.js

```diff
@@ -38,6 +38,8 @@ export function camelCaseRequestWrapper(requestFunc) {
         'bill_duration__lt', 'bill_duration__lte',
         'add_time__gt', 'add_time__gte',
         'add_time__lt', 'add_time__lte',
+        'renewal_date__gte','renewal_date__gt',
+        'renewal_date__lte','renewal_date__lte',
```

Comment:
@prabhav-plivo both should not be the lte, one filter will be the renewal_date__lte and other filter will be the renewal_date__lt
here you have made the both filter names same
cc: @kapilp93